CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ How to release a new version:
 - Manually release new version.
 
 ## [Unreleased]
+### Added
```

Comment:
Great job, this was the important missing part!
I would probably release a new version so we can use it immediately in our projects.